### PR TITLE
SREP-1751 - Expose metric with total of accounts in PendingVerification

### PR DESCRIPTION
**Pull request by rogbas**
Adding `aws_account_operator_total_aws_account_pending_verification` metric to the operator

### CODE REVIEW

File: pkg/metrics/metrics.go [hunk 1]
```diff
@@ -56,6 +56,10 @@ var (
 		Name: "aws_account_operator_pool_size_vs_unclaimed",
 		Help: "Report the difference between the pool size and the number of unclaimed account CRs",
 	}, []string{"name"})
+	metricTotalAccountPendingVerification = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "aws_account_operator_total_aws_account_pending_verification",
+		Help: "Report how many accounts are waiting for limit increase in AWS",
```
Comment (jharrington22) [inline pkg/metrics/metrics.go]: Does it not wait for the enterprise support too?

Comment (rafael-azevedo) [inline pkg/metrics/metrics.go]: Nit pick but Maybe 

"Report the number of accounts are waiting for limit increase in AWS"

Comment (rogbas) [reply pkg/metrics/metrics.go]: Yes, both. Will update the metric description

Comment (rogbas) [reply pkg/metrics/metrics.go]: updated

Comment (rogbas) [reply pkg/metrics/metrics.go]: updated

**Pull request merged**